src/utils/helpers/list/buildQueryFilters.ts

```diff
@@ -36,9 +36,8 @@ export default function buildQueryFilters(
 const filterTypeDefinitions: FilterTypeDefinitions = {
   number: (filterValue) => {
     const [fromNumber, toNumber] = filterValue.split(':')
-    const greaterThanOrEqualTo = fromNumber ? fromNumber : undefined
-    const lessThanOrEqualTo = toNumber ? toNumber : undefined
-
+    const greaterThanOrEqualTo = fromNumber !== '' ? parseInt(fromNumber) : undefined
+    const lessThanOrEqualTo = toNumber !== '' ? parseInt(toNumber) : undefined
```

Comment:
We have no way to know in the front-end whether the database field is an `Int` or a `Float`, so I'm just making sure we're only sending `Ints` in the filter object.

Reply:
It's actually enforced in the input components, this is just a last line of defence.